journey-map/public/index.html

```diff
@@ -14,7 +14,12 @@ <h1>
                     </svg>
                     <span>IPV Core Journey Map</span>
                 </h1>
-                <input id="state-search" type="search" placeholder="Search for state...">
+                <div id="header-actions">
+                    <h4>Current Journey
```

Comment:
I don't think this should be an `h4` - if it's a heading it's an `h2` (since it follows an `h1`), and otherwise it's a `label` for the select